taxi/admin.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib import admin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+# Register your models here.
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = ("username", "email", "license_number", "first_name", "last_name")
+    search_fields = ("username", "license_number")
+
+    fieldsets = (
+        ("Personal Information", {
+            "fields": ("username", "email", "first_name", "last_name"),
+        }),
+        ("Additional info", {
+            "fields": ("license_number",),
+        }),
+    )
+
+    add_fieldsets = (
+        ("Additional info", {
+            "fields": ("license_number", "first_name", "last_name"),
+        }),
+    )
```

Comment:
It would be better to inherit from UserAdmin in order not to write extra code

```suggestion
    list_display = UserAdmin.list_display + ("license_number",)
    search_fields = ("username", "license_number")
    fieldsets = UserAdmin.fieldsets + (
        ("Additional info", {"fields": ("license_number",)}),
    )
    add_fieldsets = UserAdmin.add_fieldsets + (
        ("Additional info", {"fields": ("license_number",)}),
    )
```